GDJS/Runtime/spriteruntimeobject.js

```diff
@@ -4,6 +4,62 @@
  * This project is released under the MIT License.
  */
 
+// Type Definitions
+
+/**
+ * @typedef {Object} Point Represents a Point in a frame
+ * @property {number} x X position of the point
+ * @property {number} y Y position of the point
+ */
+
+/**
+ * @typedef {Object} CustomPointData Represents a Custom Point in a frame
```

Comment:
No extra capitals: Represents a custom point in a frame

Reply:
fixed typo